CHANGES.rst

```diff
@@ -32,6 +32,8 @@ API Changes
 - Viewers now can access the calling Jdaviz application using
   ``viewer.session.jdaviz_app``. [#1051]
 
+- Viz tool display changed to ``viz.show()``. [#965]
```

Comment:
I think this should be merged with the change log for #952 . Also you need to rebase against latest `main` branch upstream (not your fork). I see the change log here has not picked up recent fixes.

Reply:
Yeah, I agree with Duy.